### ISSUE TITLE
Suggested Video Relevance is Wildly Off

### ISSUE THREAD

**Issue by prepperz** [PHPfox-Official/phpfox-v4-issues#2457]
When there is no way to add topics to videos, the displayed video suggestions will for the most display videos that have nothing to do with the topic of the video the user is watching.

**Comment by harrison05** [comment]
Could you explain more? Do you want support Topics for Videos App?

**Comment by prepperz** [comment]
Yes. Being able to include topics would help greatly. For example if you're watching a car video, you only want to see car related videos regardlesss of the category.

**Comment by harrison05** [comment]
We support hashtag and categories for Video, I think they are useful to filter Videos.
